tests/fake_http_server/fake_uptane_server.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/python
+import sys
+import socket
+import json
+from BaseHTTPServer import BaseHTTPRequestHandler, HTTPServer
+
+
+class Handler(BaseHTTPRequestHandler):
+	def do_GET(self):
+		pass
+
+
+	def do_POST(self):
+		length = int(self.headers.getheader('content-length'))
+		data = json.loads(self.rfile.read(length))
+		if self.path == "/devices":
+			return self.do_devicesRegister(data)
+		elif self.path == "/director/ecus":
+			return self.do_ecuRegister(data)
+		else:
+			self.send_response(404)
+			self.end_headers()
+
+	def do_devicesRegister(self, data):
+		if data["deviceId"] == "drop_request":
+			return
+		elif data["deviceId"] == "drop_body":
+			self.send_response(200)
+			self.end_headers()
+			self.wfile.write("some partial response")
+		elif data["deviceId"].startswith("status_"):
+			self.send_response(int(data["deviceId"][7:]))
+			self.end_headers()
+		elif data["deviceId"] == "noerrors":
+			self.send_response(200)
+			self.end_headers()
+			f = open('tests/test_data/cred.p12', 'r')
+			self.wfile.write(f.read())
+
+	def do_ecuRegister(self, data):
+		if data["primary_ecu_serial"] == "drop_request":
+			return
+		elif data["primary_ecu_serial"].startswith("status_"):
```

Comment:
I know this is getting tedious, but could we also have a drop_body case for the ECU registration?